JSDemos/Demos/Charts/Strips/description.md

```diff
@@ -0,0 +1,5 @@
+A strip is a colored piece of the **Chart**'s background that highlights a range of values.
+
+To configure strips, declare an array of objects in the **argumentAxis**.[strips[]](Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/strips/) or **valueAxis**.[strips[]](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/) property. Each object in this array configures an individual strip. The [startValue](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/#startValue) and [endValue](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/#endValue) properties define limits of the area which a strip occupies. This demo shows two strips on the value axis. We configure the [label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/)'s [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/#text) and **font**.[color](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/font/#color) properties for each strip separately.
```

Comment:
A couple of minor suggestions. I want to break this into two paragraphs. And "which" is a sign of a complicated sentence. 

```suggestion
To configure strips, declare an array of objects in the **argumentAxis**.[strips[]](Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/argumentAxis/strips/) or **valueAxis**.[strips[]](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/) property. Each object in this array configures an individual strip. The [startValue](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/#startValue) and [endValue](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/#endValue) properties define the highlight range. 

This demo shows two value axis strips. The code configures [label](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/)'s [text](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/#text) and **font**.[color](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/valueAxis/strips/label/font/#color) properties for each strip separately.
```